modules/kubernetes-addons/kubecost/values.yaml

```diff
@@ -0,0 +1,181 @@
+global:
+  grafana:
+    enabled: false
+    proxy: false
+
+pricingCsv:
+  enabled: false
+  location:
+    provider: "AWS"
+    region: "us-east-1"
+    URI: s3://kc-csv-test/pricing_schema.csv # a valid file URI
+    csvAccessCredentials: pricing-schema-access-secret
+
+nodeSelector: {}
+
+tolerations: []
+#  - key: "key"
+#    operator: "Equal|Exists"
+#    value: "value"
+#    effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
+
+affinity: {}
+
+# If true, creates a PriorityClass to be used by the cost-analyzer pod
+priority:
+  enabled: false
+  # value: 1000000
+
+# If true, enable creation of NetworkPolicy resources.
+networkPolicy:
+  enabled: false
+
+podSecurityPolicy:
+  enabled: false
+
+imageVersion: prod-1.96.0
+kubecostFrontend:
+  image: public.ecr.aws/kubecost/frontend
+  imagePullPolicy: Always
+  resources:
+    requests:
+      cpu: "10m"
+      memory: "55Mi"
+    #limits:
+    #  cpu: "100m"
+    #  memory: "256Mi"
+
+kubecostModel:
+  image: public.ecr.aws/kubecost/cost-model
```

Comment:
lets remove all under `kubecostModel` except for `image`

Reply:
actually, this might be easier. Lets start with below as the default values file

```yaml
global:
  grafana:
    enabled: false
    proxy: false

imageVersion: prod-1.96.0
kubecostFrontend:
  image: public.ecr.aws/kubecost/frontend

kubecostModel:
  image: public.ecr.aws/kubecost/cost-model

kubecostMetrics:
  emitPodAnnotations: true
  emitNamespaceAnnotations: true

prometheus:
  server:
    image:
      repository: public.ecr.aws/kubecost/prometheus
      tag: v2.35.0

  configmapReload:
    prometheus:
      image:
        repository: public.ecr.aws/bitnami/configmap-reload
        tag: 0.7.1

reporting:
  productAnalytics: false
```